lib/MultiModule.js

```diff
@@ -19,7 +20,9 @@ MultiModule.prototype = Object.create(Module.prototype);
 MultiModule.prototype.constructor = MultiModule;
 
 MultiModule.prototype.identifier = function() {
-	return "multi " + this.name;
+	return "multi " + this.dependencies.map(function(d) {
+		return path.resolve(this.context, d.request);
+	}.bind(this)).join("!");
```

Comment:
How should I update `readableIdentifier` ? I thought it's for human then `this.name` makes sense ?